funcx/executors/high_throughput/interchange.py

```diff
@@ -654,17 +681,35 @@ def start(self, poll_period=None):
                 if manager not in self._ready_manager_queue:
                     logger.warning("[MAIN] Received a result from a un-registered manager: {}".format(manager))
                 else:
+                    try:
+                        logger.debug("[MAIN] Trying to unpack ")
+                        manager_report = Message.unpack(b_messages[0])
+                        logger.info(f"[MAIN] Got manager status report: {manager_report.task_statuses}")
+                        self.task_status_deltas.update(manager_report.task_statuses)
+                        b_messages = b_messages[1:]
```

Comment:
needs to update `self._ready_manager_queue[manager]['last']` for this HB received?
I think `self._ready_manager_queue[manager]['last']` means the last message time received from a manager.

Reply:
You're right, this should be updated.  Oddly, it wasn't before either; I guess other places updated it frequently enough not to matter.

Reply:
Yes, I think previously this was only for result collection and HB update was updated somewhere else. But after moving HB here, it makes sense to update here.